metaworld/envs/mujoco/env_dict.py

```diff
@@ -52,21 +50,72 @@
 from metaworld.envs.mujoco.sawyer_xyz.sawyer_plate_slide_side import SawyerPlateSlideSideEnv
 from metaworld.envs.mujoco.sawyer_xyz.sawyer_plate_slide_back_side import SawyerPlateSlideBackSideEnv
 
+_NUM_METAWORLD_ENVS = 50
+_static_task_ids = {
```

Comment:
wait, how about you just use a tuple?

when you would have done `_static_task_ids[key]` you can just do `_static_task_ids.index(key)`

Reply:
alternatively, you can keep an OrderedDict `ALL_ENVIRONMENTS` and index into that

```py
ALL_ENVIRONMENTS = collections.OrderedDict(
    ('reach-v1', SawyerReachPushPickPlaceEnv),
    ...
    ('door-unluck-v1', SawyerDoorUnlockEnv),
)

task_id = list(ALL_ENVIRONMENTS.keys()).index(key)
```

Reply:
I'm advocating for a single source of truth here, basically.

Reply:
You could alternatively still keep a mapping from names to integers, but please use `OrderedDict`.
If you do use Ryan's suggestion (which might actually require also edding the environment args), you can make this dictionary like this:
```python
_static_task_ids = collections.OrderedDict([(k, i) for (i, k) in enumerate(ALL_ENVIRONMENTS.keys())])
```